src/nodejs_bindings.hpp

```diff
@@ -34,6 +34,7 @@ class Annotator final : public Nan::ObjectWrap
     static Nan::Persistent<v8::Function> &constructor();
 
     /* Wrapping Annotator; both database and annotator do not provide default ctor: wrap in ptr */
+    bool createRTree;
```

Comment:
Maybe `= false;` here, too. Otherwise this member attribute is uninitialized until you write to it.